t/50access-log-transport.t

```diff
@@ -0,0 +1,98 @@
+use strict;
+use warnings;
+use Net::EmptyPort qw(empty_port wait_port);
+use File::Temp qw(tempdir);
+use JSON;
+use Test::More;
+use t::Util;
+
+my $tempdir = tempdir(CLEANUP => 1);
+
+my $client_prog = bindir() . "/h2o-httpclient";
+plan skip_all => "$client_prog not found"
+    unless -e $client_prog;
```

Comment:
Now that we depend on the mruby handler, please skip the tests `unless server_features()->{mruby}`.